src/ducks/stripe.duck.js

```diff
@@ -627,8 +627,12 @@ export const handleCardPayment = params => dispatch => {
 
   dispatch(handleCardPaymentRequest());
 
+  const args = card
```

Comment:
Might be useful to add code comment that Stripe Element is optional and it's handled here